stable/polaris/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Validation of best practices in your Kubernetes clusters
 name: polaris
-version: 4.2.3
-appVersion: "4.2"
+version: 5.0.0
```

Comment:
This doesn't seem like a major change to the chart. Honestly it feels more like a patch version to me